### ISSUE TITLE
Remove text clipping on typeahead autosuggestions

### ISSUE THREAD

**Issue by cjg89** [UCF/UCF-Announcements-Django#59]
https://github.com/UCF/UCF-Announcements-Django/blob/b8b2afdf2fe5c9eaa12eb86b2d1c0b6ba8934154/src/scss/_forms.scss#L12-L18

`overflow: hidden;`, `text-overflow: ellipsis;`, and `white-space: nowrap;` should be removed from the snippets linked above to ensure that users zoomed up to 200% will not have suggestion contents clipped that users at a standard zoom level would be able to see.

See also https://www.w3.org/WAI/WCAG21/Understanding/resize-text
